zerver/lib/test_classes.py

```diff
@@ -950,13 +949,25 @@ def assert_json_success(self, result: HttpResponse) -> Dict[str, Any]:
         return json
 
     def get_json_error(self, result: HttpResponse, status_code: int = 400) -> str:
+        json = self.assert_json_error_data(result, status_code=status_code)
+        return json["msg"]
+
+    def assert_json_error_data(
+        self, result: HttpResponse, status_code: int = 400
+    ) -> Dict[str, Any]:
+        """
+        Assert response has the given status code. It assume the JSON of the form
+        {"result": "error", "msg": msg} when `use_zulip_error_format` is `True`.
```

Comment:
Finish removing `use_zulip_error_format`.

Reply:
I removed `assert_json_error_data` and restored the original `.json` calls. The next step will be applying `_MonkeyPatchedWSGIResponse` to wrap other uncovered cases of `HttpResponse` up.